### PR TITLE
Fixed registration

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: Server/VikingVault.DataAccess/Migrations/20190813074007_DroppedProfilePages.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VikingVault.DataAccess.Migrations
+{
+    public partial class DroppedProfilePages : Migration
```
Comment (vasi-axinte) [inline Server/VikingVault.DataAccess/Migrations/20190813074007_DroppedProfilePages.cs]: Is this Migration related to the registration problem?

**Review by vasi-axinte — CHANGES_REQUESTED** [review]

File: Server/VikingVault.DataAccess/Migrations/20190813074007_DroppedProfilePages.cs (hunk 1 above)
Comment (robertpin) [reply Server/VikingVault.DataAccess/Migrations/20190813074007_DroppedProfilePages.cs]: No. I removed it

**Review by vasi-axinte — APPROVED** [review]
